demos/adventure/README.md

```diff
@@ -0,0 +1,97 @@
+# Adventure Demo
+
+## What does it do?
+* It applies various different things from multiple engo demos
+* It demonstrates how one can load and render a TMX file created from the TileMap Editor
+* It shows how to add a character entity
+  * It demonstrates how to animate the character and move it around
```

Comment:
If by weird tab you mean the indented line 7, this is intentional. It's GitHub markdown syntax.
Line 7 is a sub point of line 6, so it'll look like this:

* It shows how to add a character entity
  * It demonstrates how to animate the character and move it around

The reason why I chose to do this here, is to make clear to the user that the character animation part is a sub-topic of how to add a character entity in the first place.